pipelines/parsing.pipeline.json

```diff
@@ -21,9 +21,8 @@
   "input": {
     "pfs": {
       "repo": "documents",
-      "glob": "/*"
-    }
-  },
+      "glob": "/"
```

Comment:
same here. with a glob of `/` you're limited to a single worker and processing them sequentially.

Reply:
I added /* back